### ISSUE TITLE
Suggestion: Disable score voting from the search page with the "Show scores" option

### ISSUE THREAD

**Issue by luni3359** [danbooru/danbooru#4932]
Allowing users to rate posts based on the small thumbnails from in the front page doesn't feel right. There's a huge amount of detail that cannot otherwise be appreciated at that size. The "Show scores" button should only show scores without presenting the options to like or dislike.

**Comment by luni3359** [comment]
To add on this, it'd make more sense if the bottom of each post thumbnail had favorite and score counts instead.

**Comment by BrokenEagle** [comment]
I'd be against doing anything like this, especially when such would be still easy to do with a script.  Besides, I'd say at least ~3/4 of all images I evaluate can be accurately evaluated on their merits, especially since most images are 1 person solo, using either the portrait, upper body, or cowboy shot framing. For those that are too hard to distinguish, I open up the post in a new tab and make my evaluations there. However, asking me and every other user to have to open a new tab for every single image we want to evaluate would be nothing short of draconian, and imagine it would be vastly unpopular with a majority of the userbase.

**Comment by luni3359** [comment]
Nobody is asking you to do it. The previous way to achieve this was through the scripts dropdown, and even then I doubt any more than fringe users ever made use of it.

I'm going based on the behavior of other image-browsing sites/applications, like Pixiv and Imgur. Their thumbnails are larger, and they crop zoom-in their pictures too thus giving your point on not opening a post more value, but this isn't done on desktop Danbooru (barring the triviality of changing the css). 

On mobile the difference is even noticeable, with both of the sites I mentioned limiting their display of images to two big rows that can be appreciated more clearly. You can almost fit one pixiv thumbnail in the space of four danbooru thumbnails. With the option showing scores enabled it is also a little hard to tap the button that you intend to. You might end up tapping either the post above or below.

I am putting my thoughts out on what I believe will benefit the masses more than it will a small group of people that are used to working on things in one specific way.

**Comment by luni3359** [comment]
tl;dr I believe it's okay to keep the buttons only if the thumbnails were larger, but they're not.

**Comment by BrokenEagle** [comment]
I would be fine with having a larger thumbnail size available, though still not having that contingent on whether the vote option should be available or not for the same reasons I've already mentioned.

To be honest, 150px is a dinosaur relic from when this site was first started in 2007. Monitors have gotten a lot better and a lot bigger since then. Some users already use userscripts which substitute the thumbnail size for the sample size taking up more server bandwidth, whereas they probably wouldn't do that if a larger thumbnail size was available.

If needed the old thumbnail size could be preserved for those that would like for them to still be small. It would be much like Windows file previews for images, where there is a large size and an extra-large size.

My own idea would be to set the thumbnail dimensions to 300px. I already do this on a Danbooru-like project that I've been working on.

![image](https://user-images.githubusercontent.com/21149935/143621410-156ceacd-58d5-42c7-9b7b-c78d35534946.png)

While I'm on the subject, the sample size could potentially use updating as well. Wide images take the worst abuse with this old aspect. On the same project I mentioned above, I use 1280W x 1920H for the max dimensions, which is the primary sample size for other sites like Tumblr. Since that can make some images flow outside the screen, I set it to start out scaled down to 100VH so that the entire image fills up screen, and have it alternate between full sample size and screen size by clicking the image. Since image clicks are used for notes on Danbooru, this could be substituted with a hotkey or something.

**Comment by luni3359** [comment]
Yeah, I agree with everything you said. That means that the real issue I'm seeing is more about the size of the thumbnails like you mentioned. I'm going to change the title, but it shouldn't change the original intent.

**Comment by evazion** [comment]
After experimenting with this a lot, I'm settling on 180x180 and 360x360 thumbnails, with options to select between small, medium, and large sizes.

180x180 divides most common screen resolutions evenly. It fits two thumbnails per row on phones (where the most common resolution is 360x640), and six or eight per row on most laptops or desktops (after factoring in the sidebar).

360x360 allows for both bigger thumbnails on desktop, and higher quality thumbnails on phones and laptops with high DPI displays. On desktop, 360x360 is a bit large for regular browsing (it only gives 4 thumbnails per row at 1920x1080), but you can downscale it in the browser to 225x225 or 270x270 to get medium and large size thumbnails. On phones, 360x360 can be used for either full width thumbnails, or 180x180 thumbnails with 2x pixel density, which gives much higher quality images.

>While I'm on the subject, the sample size could potentially use updating as well. Wide images take the worst abuse with this old aspect. On the same project I mentioned above, I use 1280W x 1920H for the max dimensions, which is the primary sample size for other sites like Tumblr.

I'll think about this after handling thumbnails. We probably need at least three sample sizes, for phones, laptops/tablets, and desktops. Probably more than that, for high DPI displays.

>Since that can make some images flow outside the screen, I set it to start out scaled down to 100VH so that the entire image fills up screen, and have it alternate between full sample size and screen size by clicking the image. Since image clicks are used for notes on Danbooru, this could be substituted with a hotkey or something.

This is another thing I've wanted to do for a long time, but I've held off because I expect people won't like it at first, even though it's a really necessary change.

**Comment by BrokenEagle** [comment]
Does this need to also have a fix script?

**Comment by GlassedSilver** [comment]
> Does this need to also have a fix script?

I'm pondering the same.

My local instance has been looking a bit off these past few weeks when I upgraded it.

I haven't gotten around to fix it yet, at least I got to look into the issue today and figure out what caused this. :)

Small progress in the end and at least I know I'm not suffering data loss without realizing. :D
